examples/api-dotenv-request/test/Spec.hs

```diff
@@ -0,0 +1,2 @@
+main :: IO ()
+main = putStrLn "Test suite not yet implemented"
```

Comment:
We can remove the test suite from the cabal project and then remove this file along with the directory.